rllib/algorithms/appo/torch/appo_torch_rl_module.py

```diff
@@ -42,8 +32,26 @@ def output_specs_train(self) -> List[str]:
 
     @override(PPOTorchRLModule)
     def _forward_train(self, batch: NestedDict):
+        if self.inference_only:
```

Comment:
nit: Maybe we should move this error into the parent RLModule class' main method?
`def forward_train(self, ...):`?